randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1280,10 +1305,14 @@ Extra - asset_id: collision_camera_019
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Door to EMMI Zone Exit Northeast (doorpowerpower_010)
-      Flash Shift or Space Jump
+  > Door to EMMI Zone Exit Northeast (Upper)
+      Any of the following:
+          Flash Shift or Space Jump
+          Slide and Slide Jump (Beginner) and Use Spin Boost
   > Room Center
       Trivial
+  > Pickup (Ice Missile)
+      After Ghavoran - Central Unit
```

Comment:
What's the reasoning behind getting the ice missile pickup from the northeast door as opposed to the room center?

Reply:
First bullet in the description.

"The Ice Missile pickup is now accessed from the door to EMMI Zone Exit West since it is not possible to defeat the EMMI on the ground floor, and to defeat it you would have to have gone up here to either defeat it here or on the other side"

Reply:
Gotcha